### PR TITLE
SOC-532 Convert the founder tips emails to use the new templates and email extension

### PR DESCRIPTION
**Pull request by jsutterfield**
Ticket: https://wikia-inc.atlassian.net/browse/SOC-532

### CODE REVIEW

File: extensions/wikia/Email/Controller/FounderController.class.php [hunk 1]
```diff
@@ -449,3 +449,225 @@ public function assertFounderWantsNewMembersEmail() {
...
+
+	/**
+	 * Returns list of details (icons, headers, and blurbs for those icons) for the founder tips email
+	 *
+	 * @return array
+	 */
+	protected function getDetailsList() {
+		return [
+			[
+				"iconSrc" => Email\ImageHelper::getFileInfo( "Add_page.png" )['url'],
```
Comment (garthwebb) [inline extensions/wikia/Email/Controller/FounderController.class.php]: What do you think about adding a ```getFileUrl``` method to ```Email\ImageHelper```?  Seems helpful.

File: extensions/wikia/Email/i18n/Founder.i18n.php [hunk 2]
```diff
@@ -45,6 +80,41 @@
...
+	'emailext-founder-10-days-sharing-header' => "Header for the section which encourages the founder to share their wiki on social media using the share buttons",
+	'emailext-founder-10-days-sharing-details' => "Line encouraging the founder to share their wiki on social media using the share buttons",
+	'emailext-founder-10-days-email-power-header' => "Header for the section encouraging the founder to email users to get them more involved and active on the wiki",
+	'emailext-founder-10-days-email-power-details' => "Line encouraging the founder to email users to get them more involved and active on the wiki",
+	'emailext-founder-10-days-email-google-header' => "Header link which gives the founder tips on improving their SEO. See 'emailext-founder-get-with-google'",
+	'emailext-founder-10-days-email-google-details' => "Line which reminds the user the internet is vast and they should work on improving their SEO to draw more traffic",
+	'emailext-founder-get-with-google' => "Link to page on community.wikia.com which gives tips on helping with SEO.",
+	'emailext-founder-10-days-email-what-next' => "Line which tells the user we're excited to see what they're going to do with their wiki",
```
Comment (garthwebb) [inline extensions/wikia/Email/i18n/Founder.i18n.php]: I think you've really found your voice in these qqq's

File: extensions/wikia/FounderEmails/events/FounderEmailsDaysPassedEvent.class.php [hunk 3]
```diff
@@ -1,159 +1,107 @@
...
-					$dayName = 'DayTen';
-					$doProcess = false;
-					break;
-				default:
-					$dayName = 'DayZero';
-					$doProcess = true;
-			}
-
-			$mainPage = wfMsgForContent( 'mainpage' );
+		foreach ( $wgFounderEmailsExtensionConfig['events']['daysPassed']['days'] as $activateDay ) {
```
Comment (garthwebb) [inline extensions/wikia/FounderEmails/events/FounderEmailsDaysPassedEvent.class.php]: Same as above, and strike the ```'events'``` key usage.  Should be same as previous version, ```$wgFounderEmailsEvents['daysPassed']['days']```

File: extensions/wikia/Email/i18n/Founder.i18n.php [hunk 4]
```diff
@@ -22,6 +22,41 @@
...
+	'emailext-founder-3-days-add-videos-details' => "Make your page awesome and interesting by adding your favorite fandom videos. And who doesn’t love a good video?",
+	'emailext-founder-3-days-update-theme-header' => "Update Your Theme",
+	'emailext-founder-3-days-update-theme-details' => "Themes, colors, wordmarks, oh my! Use the Wikia Theme Designer to create a design that matches your fandom perfectly.",
+	'emailext-founder-3-days-wam-header' => "Get Inspired",
+	'emailext-founder-3-days-wam-details' => "Draw inspiration from some of our [$1 favorite communities]. These represent the best and most popular wikia fan sites!",
+	'emailext-founder-3-days-need-help' => "Need help figuring out how something works? We're always here for you! Ask us for help and advice at [http://community.wikia.com Community Central].",
+	'emailext-founder-3-days-great-work' => "Keep up the great work! ",
+	'emailext-founder-10-days-subject' => "Happy 10-day anniversary on $1!",
+	'emailext-founder-10-days-summary' => "It’s been 10 days since you started $1.",
+	'emailext-founder-10-days-extended-summary' => "We must say, your wikia are looking great.  Now it’s time to spread the word!",
```
Comment (mklucsarits) [inline extensions/wikia/Email/i18n/Founder.i18n.php]: "your wikia are looking great" seems wrong. Can you double-check the copy for this message?

File: extensions/wikia/Email/i18n/Founder.i18n.php (hunk 2 above)
Comment (jsutterfield) [inline extensions/wikia/Email/i18n/Founder.i18n.php]: Thank you for recognizing that. It's been a long, hard road, but I think it's finally starting to pay off.

File: extensions/wikia/Email/Controller/FounderController.class.php (hunk 1 above)
Comment (jsutterfield) [inline extensions/wikia/Email/Controller/FounderController.class.php]: I think that is a fine idea.

File: extensions/wikia/Email/i18n/Founder.i18n.php (hunk 4 above)
Comment (jsutterfield) [inline extensions/wikia/Email/i18n/Founder.i18n.php]: Good catch. That's not good. I'll updated.

**Comment by mklucsarits** [comment]
LGTM

**Pull request merged**